code/game/gamemodes/blob/overmind.dm

```diff
@@ -36,14 +36,17 @@
 	updateallghostimages()
 	..()
 
+/mob/camera/blob/Life()
+	if(!blob_core)	Destroy()

```

Comment:
1. Pretty sure calling Destroy() is wrong, just qdel(src)
2. Don't write shit like this one 1 line.